addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -6416,7 +6419,7 @@ registry.ImageTools = ImageHandlerOption.extend({
             imgAspectRatio: svg.dataset.imgAspectRatio || null,
             svgAspectRatio: svgAspectRatio,
         };
-        const imgDataURL = await applyModifications(img, options);
+        const { dataURL: imgDataURL } = await applyModifications(img, options);
```

Comment:
No need to use the output mimetype here as the dataURL is not used in an img attribute, but rather inlined in an svg. The right mimetype is included directly in the dataURL and the actual output of this method is always an SVG